docs/data/material/components/text-fields/InputSuffixShrink.js

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { outlinedInputClasses } from '@mui/material/OutlinedInput';
+import { filledInputClasses } from '@mui/material/FilledInput';
+import { inputBaseClasses } from '@mui/material/InputBase';
+import TextField from '@mui/material/TextField';
+import InputAdornment from '@mui/material/InputAdornment';
+
+export default function InputSuffixShrink() {
+  return (
+    <Box
+      component="form"
+      sx={{ '& > :not(style)': { m: 1, width: '25ch' } }}
+      noValidate
+      autoComplete="off"
+    >
+      <TextField
+        id="outlined-basic"
```

Comment:
Clicking this demo's labels moves focus to the inputs in the first demo; we should use different ids.

<img width="852" alt="image" src="https://github.com/user-attachments/assets/07b1fa76-db0d-47ca-bd11-9d29f2439a69" />

Reply:
Nice catch